### PR TITLE
Add vg clothing to the vending machines

### PR DESCRIPTION
**Pull request by TaxMaster**
Adds some of the VG clothing to the clothesmate and autodrobe vending machines along with updating the the refill packs.

### CODE REVIEW

File: code/game/machinery/vending.dm [hunk 1]
```diff
@@ -916,8 +916,50 @@ IF YOU MODIFY THE PRODUCTS LIST OF A MACHINE, MAKE SURE TO UPDATE ITS RESUPPLY C
...
+					/obj/item/clothing/shoes/joeboots = 1,
+					/obj/item/clothing/gloves/joegloves = 1,
+					/obj/item/clothing/head/helmet/joehelmet = 1,
+					/obj/item/clothing/under/joe = 1,
+					/obj/item/clothing/under/vault = 1,
+					/obj/item/clothing/under/roll = 1,
+					/obj/item/clothing/head/clownpiece = 1,
+					/obj/item/clothing/under/clownpiece = 1,
+					/obj/item/clothing/suit/clownpiece = 1)
+	contraband = list(/obj/item/clothing/suit/judgerobe = 1,/obj/item/clothing/head/powdered_wig = 1,/obj/item/weapon/gun/magic/wand = 2,/obj/item/clothing/glasses/sunglasses/garb = 2,
```
Comment (Poojawa) [inline code/game/machinery/vending.dm]: This needs to be tabbed better

**Comment by TaxMaster** [comment]
Alright, is this better?

**Comment by TaxMaster** [comment]
Updated the vending machine list based on what Cactus said.

**Comment by TaxMaster** [comment]
Because honestly, in the roughly one month span that the clothing has been in the vending machine, I didn't see or hear of a single person complaining about any of the added clothing.

**Pull request merged**